code/classes/DMSDocumentCart.php

```diff
@@ -21,11 +21,23 @@ public function __construct($backend = null)
     /**
      * Returns all the cart items as an array
      *
-     * @return array
+     * @return ArrayList
      */
     public function getItems()
     {
-        return $this->backend->getItems();
+        $result = $this->backend->getItems();
+
+        return $result;
+    }
+
+    /**
+     * Returns all the cart items as an ArrayList
+     *
+     * @return ArrayList
+     */
+    public function getItemsAsArrayList()
+    {
+        return ArrayList::create($this->getItems());
```

Comment:
This was a convenience method as I found I needed an ArrayList very often. getItems() was already written and used in other places so I opted for this instead to shorten code in places where one is needed `$arrayList = $this->getItemsAsArrayList())` is much shorter and safer than doing `$arrayList = ArrayList::create($this->getItems())` all the time.